scripts/generate_learning_path_markdown.js

```diff
@@ -1,14 +1,18 @@
 const { join, basename, relative } = require('path')
+const fs = require('fs')
+const yamlFront = require('yaml-front-matter')
+
 const getContributors = require('./get_contributors')
 const mkdirSync = require('./mkdir_sync')
 const getArticleFiles = require('./get_article_files')
+const writeMarkdownFile = require('./write_markdown_file')
 
 const sections = require('./section_data.json')
 
-module.exports = async (writeDir, generatorFn, workbookFn, createTranslationFolder) => {
+module.exports = async (writeDir, generatorFn, workbookFn, isJekyllSite) => {
```

Comment:
Better variable naming/purpose so as not to bleed implementation details out of this function